01-filedir.md

```diff
@@ -108,45 +123,47 @@ Inside that directory are several other directories:
 `tmp` (for temporary files that don't need to be stored long-term),
 and so on:
 
-![The File System](fig/filesystem.svg)
-
 We know that our current working directory `/Users/nelle` is stored inside `/Users`
 because `/Users` is the first part of its name.
 Similarly,
 we know that `/Users` is stored inside the root directory `/`
 because its name begins with `/`.
 
-Underneath `/Users`,
-we find one directory for each user with an account on this machine.
-The Mummy's files are stored in `/Users/imhotep`,
-Wolfman's in `/Users/larry`,
-and ours in `/Users/nelle`,
-which is why `nelle` is the last part of the directory's name.
-
-![Home Directories](fig/home-directories.svg)
-
-> ## Path {.callout}
+> ## Slashes {.callout}
 >
 > Notice that there are two meanings for the `/` character.
 > When it appears at the front of a file or directory name,
 > it refers to the root directory. When it appears *inside* a name,
 > it's just a separator.
 
-Let's see what's in Nelle's home directory by running `ls`,
+Underneath `/Users`,
+we find one directory for each user with an account on Nelle's machine, 
+her colleagues the Mummy and Wolfman.  
+
+![Home Directories](fig/home-directories.svg)
+
+The Mummy's files are stored in `/Users/imhotep`,
+Wolfman's in `/Users/larry`,
+and Nelle's in `/Users/nelle`.  Because Nelle is the user in our 
+examples here, this is why we get `/Users/nelle` as our home directory.  
```

Comment:
Maybe a note that this is generally the directory where you start when you open a shell?